### ISSUE TITLE
Link portlet - Be able to save at the first step when I edit the portlet

### ISSUE THREAD

**Issue by Julien-Dubois-eXo** [Meeds-io/meeds#1280]
**Is your feature request related to a problem? Please describe.**
When I'm editing a portlet link and I just want to modify the links I have to go to the step 2 even if I don't want to edit it to have the save button active.

Presently it's painful.

**Describe the solution you'd like**
When I'm not editing for the first time the portlet and I have already sets the second step, it could be nice to have the save button active even at the first step. 

**Describe alternatives you've considered**

**Additional context**

**Comment by srenault-meeds** [comment]
> When I'm not editing for the first time the portlet and I have already sets the second step, it could be nice to have the save button active even at the first step.

How does the platform know you don't want to add links?
In addition to this, we could add a plus icon instead of cogs icon to allow only add links and not reviewing the display. But I have the feeling that this will add complexity to the end-user


Indeed, this feedback is interesting but I don't have for now an answer.
So for now, I put it on waiting on status

**Comment by Julien-Dubois-eXo** [comment]
The issue is we use the same location to put the "Next" button and the "save" button. If the locations are different we can just activate the "save" button if a modification is done no matter the steps we are in. The platform does not need to know what change we want to do.
